x/profile/types/errors.go

```diff
@@ -12,7 +12,6 @@ var (
 	ErrCoordAddressNotFound      = sdkerrors.Register(ModuleName, 3, "coordinator address not found")
 	ErrCoordInvalid              = sdkerrors.Register(ModuleName, 4, "invalid coordinator")
 	ErrEmptyDescription          = sdkerrors.Register(ModuleName, 5, "you must provide at least one description parameter")
-	ErrValidatorNotFound         = sdkerrors.Register(ModuleName, 6, "validator address not found")
 	ErrInvalidValidatorSignature = sdkerrors.Register(ModuleName, 7, "invalid validator signature")
```

Comment:
We should ensure the codespace numbers are going in ascending order.  Now that `ErrValidatorNotFound` is removed, we jump from 5 to 7

Reply:
Thanks. I don't think it is a problem technically speaking but that's better for consistency